elements/pf-table/pf-table.css

```diff
@@ -0,0 +1,15 @@
+:host {
+  display: grid;
+  border: none;
+}
+
+@media (min-width: 768px) {
+  :host {
+    display: table;
```

Comment:
The approach taken here is really interesting and I think has merit in allowing us to utilize shadow DOM to isolate and control stying.

That being said, and if we take this tack, we should consider using css-grid in place of table modes. I haven't found an effective means of emulating colspan with table modes, and we'll need that for expanded rows; whereas with grid, we can just `grid-column: 1/-1` and call it a day.

Reply:
In gchat, I'll share with you some initial progress towards implementing expanded rows for the project we mentioned earlier there.